app/assets/javascripts/tylium/behaviors.js.coffee

```diff
@@ -61,6 +61,9 @@ document.addEventListener "turbolinks:load", ->
   # Activate jQuery.Textile
   $('.textile').textile()
 
+  # Activate Editor Toolbar after Textile
+  new EditorToolbar($('[data-id~=textile-inner], [data-behavior~=comment-feed]'));
```

Comment:
This seems like a "hard-coded" approach? It's not a behaviour that we sprinkle around the DOM, but a hard-coded list of elements we want to apply the behaviour to.

I'd say `data-behaviour="rich-editor"` and then apply that anywhere we want it? Then we can make the component that renders "Write/Form-View" to include that behaviour in the rendered items it manages.

Reply:
Yup this pattern sounds familiar 😄 

Updated with new changes.